src/dialog-user/services/dialogData.ts

```diff
@@ -143,4 +148,31 @@ export default class DialogDataService {
 
     return validation;
   }
+
+  /**
+   * Determines if a value is a tag control and whether or not that value is valid
+   * @memberof DialogDataService
+   * @function isInvalidTagControl
+   * @param forceSingleValue {boolean} Whether or not the field allows multiple selections
+   * @param fieldValue {any} This is the value of the field in question to be validated
+   **/
+  private isInvalidTagControl(forceSingleValue, fieldValue) {
+    let invalid = false;
+
+    if (forceSingleValue) {
+      if (_.isNumber(fieldValue)) {
+        if (fieldValue === 0) {
+          invalid = true;
+        }
+      } else if (_.isEmpty(fieldValue)) {
+        invalid = true;
+      }
+    } else {
```

Comment:
Remove the else but keep the if statement that is inside this.  Same logic, just cuts down on a few lines

Reply:
It's not the same logic though, because `_.isEmpty` will return true when the value that gets passed in is a number, which is wrong.